client/components/jetpack/connection-health/index.tsx

```diff
@@ -63,6 +64,20 @@ export const JetpackConnectionHealthBanner = ( { siteId }: Props ) => {
 		);
 	}
 
+	if ( errorType === DATABASE_ERROR ) {
+		return (
+			<ErrorNotice
+				errorType={ errorType }
+				errorText={ translate( 'Jetpack can’t establish a connection to your site’s database.' ) }
+				noticeActionHref={ localizeUrl(
+					'https://wordpress.com/support/why-is-my-site-down/#theres-an-issue-with-your-sites-jetpack-connection'
```

Comment:
I created an issue to add an explanation to Resolve Jetpack Errors: https://github.com/Automattic/en.support-docs-content/issues/2080